oeps/processes/oep-0063-proc-toc-appeal.rst

```diff
@@ -0,0 +1,110 @@
+
+However, some of the decisions can sometimes be complex: lack of consensus, decisions with long-lasting reprecussions. This can result in delaying taking *any* decision, which can be more hurtful than any of the choices.
+
+In this OEP, the TOC defines a process for community members to bring up topics to the board's attention, and request to take a decision about a proposal (an "Appeal"). 
+
+Motivation
+**********
+
+The TOC discussed during the April 2023 meeting a `series of guidelines for operating and taking decisions <https://discuss.openedx.org/t/2023-04-11-toc-meeting-summary/10019#governance-norms-5>`_. This includes establishing a way for the community to use the TOC to help take decisions, which this OEP implements.
+
+Guidelines
+==========
+
+From the `TOC meeting notes <https://discuss.openedx.org/t/2023-04-11-toc-meeting-summary/10019#governance-norms-5>`_:
+
+*The TOC should not operate “in the weeds.” We don’t have the time to be involved day-to-day technical decisions, and those closest to the work will make the best decisions. The TOC should not act as a “stage gate.” We should not create friction for contributors unnecessarily, but should:*
+   
+- *Guide the project toward shared goals*
+- *Help refine and improve plans and designs*
+- *Prevent the project from making bad decisions that are hard to reverse.*
+   
+*The TOC should weigh in:*
+   
+- *on “one-way-doors,” decisions that are hard to reverse, where choices are mutually exclusive*
+- *when the community is having difficulty reaching consensus*
+- *when an important decision is being deferred and not making a call is causing friction*
+- *to focus the community to collaborate on key, shared value, say, investment in maturing LTI*
+
+Specification
+*************
+
+- Appeals can be posted by anyone in the community - but need to be supported by at least two core contributors or a TOC member
+- The TOC members are encouraged to participate in community discussions early on, before the TOC is asked to take the final decision.
+
+Step-by-step - How to submit an appeal
+======================================
+
+- Open a discussion in a public place, such as the `forum <discuss.openedx.org/>`_ or using one of the formal decision recording formats such as `OEPs <https://open-edx-proposals.readthedocs.io/en/latest/>`_. The topics presented need to have been discussed in the community before being formally considered by the TOC.
+- After public review, post a formal request to appeal to the TOC. This can be done in a forum post or a `github ticket <https://github.com/openedx/wg-coordination/issues/new>`_ mentioning the TOC chair (@e0d). See the format below.
+- Core contributors and/or TOC members supporting the appeal reply in the thread to say so
+- If the requirements are met, the TOC will schedule the topic for a future meeting. 
```

Comment:
@e0d Good point - I'm guessing the TOC will still want to have the possibility to do some of the discussion behind closed doors (though there would still be the summary notes published as usual, so it's not fully closed doors), it makes sense for the person who is submitting the request to be able to present it. And if that's a discussion with multiple parties (for example if there is a disagreement on a direction to take), to be fair the presentation would probably have to include the different parties/opinions. 

To also keep it async, maybe it could be under the format of being able to submit presentation videos/materials for the TOC to review async? Any party who would like to could submit a video/presentation async in advance of the meeting. The TOC members, when reviewing the materials in advance, could request some of the parties to join the meeting, to ask questions or discuss.

Let me know what you think? If that works for you, I'll add it to the document.